content/articles/improving-conversation-with-the-perspective-api/index.md

```diff
@@ -0,0 +1,127 @@
+
+### The WordPress Approach
+Making use of the Toxic Comments WordPress extension is a fantastic strategy to ensure that the associated work is completed as efficiently as possible. The use of this module lessens the necessity for dreary errands, which is a fabulous depiction of the work that the module acts in the background to help us with achieving our goals.
+
+```php
+$request = wp_remote_post($arguments, $url);
+```
+
+A request to an external asset can be published using this manner without needing an incredible amount of effort on the part of the author. We don't need to think about it right now, but there are other capabilities that you can use, such as the ability to receive requests, that we can take advantage of in the future. To obtain the information you requested from the server, you will need to use a separate function. As it turns out, WordPress does indeed provide a tool that can assist you in accomplishing this:
+
+```php
+$data = wp_remote_retrieve_body($request);
+```
+
+How can we make use of the API to obtain the information that we require? A thorough toxicity rating can only be acquired by requesting the API to read and score the comment by using the URL provided in combination with the comment and submitting the request. You'll also receive your API key at the conclusion of the process, which you'll need in order to authenticate any queries you make. It is necessary to mention your name and address.
+
+https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=YOUR-API-KEY
+
+There is nothing complicated about it at first glance, but when you try to access it, you are redirected to the 404 error page instead. It will respond to a cURL request, whether made using your chosen CMS or directly through a PHP script. The data returned will be in the following format:
+```JSON
+{
+  "attributeScores": {
+    "TOXICITY": {
+      "summaryScore": {
+        "value": 0.567890,
+        "type": "PROBABILITY"
+      }
+    }
+  },
+  "languages": [
+    "en"
+  ]
+}
+```
+As a result of your request, you will receive a decimal number as the final result. The API will then return for that comment a score of 0.5, which indicates that the comment has a 50% chance of being toxic. It is possible to use this score to govern how a comment is handled and displayed to the end user by labeling it as spam or offering a channel that allows clients to show less or more negative comments, just as Google has done with their method of scoring.
+
+![Image-1](Image-1)

```

Comment:
Please change all image path to this format 
`![Allcar] (/engineering-education/introduction-to-unity-engine /allcar.png)`